app/admin.py

```diff
@@ -1,13 +1,14 @@
 from django.contrib import admin
-from .models import User, Stack, SpokenLanguage, Request, InterestedMentor
+from .models import User, Skill, SpokenLanguage, Request, RequestInterest
 
 
-class UserAdmin (admin.ModelAdmin):
-    readonly_fields = ('user_id',)
+# class UserAdmin (admin.ModelAdmin):
+#     readonly_fields = ('user_id',)
```

Comment:
Allowed me display user_id in the django-admin page (before I got started with the API). Commented it out when I found I didn't need it anymore since it shows up because of the serializer.

Reply:
mm, not yet though. Maybe when we're all done with the backend?

Reply:
You can always add it back, or make a note to not forget about it though. Your call :)